app/client/views/admin/users/adminUsersCtrl.js

```diff
@@ -117,12 +118,13 @@ angular.module('reg')
               closeOnConfirm: false
               }, function(){
 
-                UserService
+               UserService
                   .admitUser(user._id)
-                  .success(function(user){
+                  .success(function(user) {
                     $scope.users[index] = user;
                     swal("Accepted", user.profile.name + ' has been admitted.', "success");
-                  });
+                    AuthService.sendAcceptanceEmail(user.email);
```

Comment:
One of the parameters of the sendAcceptanceEmail is a callback. If that method fails, we should know about it (the method should fail gracefully). I know this is a bit nitpicky, but unexpected issues could come up in production, and we'll spend more time debugging the crash. This way we'll know exactly what went wrong. I'd recommend adding a function that logs it somewhere.

Reply:
Also, I think the email acceptance method doesn't necessarily need to live in AuthService, you can modify it just to live in the UserService where it's invoked in admitUser automatically or through another method in UserService.

Reply:
Keeping it in UserService stays consistent with the rest of the app layout. AuthService should keep its scope where it is without adding it in other places.